### ISSUE TITLE
can I use this software without GPU(only CPU)? if yes which form have the command-calling? thanks

### ISSUE THREAD

**Issue by stefanache** [zhwzhong/Guided-Depth-Map-Super-resolution-A-Survey#4]

**Comment by stefanache** [comment]
in fact I tried that but give me the error:
1.Train
'nvidia-smi' is not recognized as an internal or external command,
operable program or batch file.

PS: the command was:
python C:\development\python\depth_map\Guided-Depth-Map-Super-resolution-A-Survey\code\main.py  --scale=8 --dataset_name='NYU' --model_name=dgf

**Comment by stefanache** [comment]
then I inspect command forms:
usage: main.py [-h] [--cpu CPU] [--num_gpus NUM_GPUS]
               [--num_workers NUM_WORKERS] [--cuda_name CUDA_NAME] [--pre_up]
               [--data_max DATA_MAX] [--data_mean DATA_MEAN]
               [--val_ratio VAL_RATIO] [--train_ratio TRAIN_RATIO]
               [--data_range DATA_RANGE] [--data_augment DATA_AUGMENT]
               [--dataset_name DATASET_NAME] [--down_type DOWN_TYPE]
               [--down_direction DOWN_DIRECTION] [--test_set TEST_SET]
               [--num_res NUM_RES] [--with_noisy] [--noisy_level NOISY_LEVEL]
               [--test_only] [--lr LR] [--optimizer OPTIMIZER]
               [--weight_decay WEIGHT_DECAY] [--lr_scheduler LR_SCHEDULER]
               [--lr_decay_rate LR_DECAY_RATE] [--patience PATIENCE]
               [--warmup_epoch WARMUP_EPOCH]
               [--warmup_multiplier WARMUP_MULTIPLIER]
               [--lr_decay_epochs LR_DECAY_EPOCHS] [--show_every SHOW_EVERY]
               [--scale SCALE] [--num_epochs NUM_EPOCHS]
               [--num_features NUM_FEATURES] [--in_channels IN_CHANNELS]
               [--guide_channels GUIDE_CHANNELS] [--pyramid_dir PYRAMID_DIR]
               [--act ACT] [--norm NORM] [--batch_size BATCH_SIZE]
               [--patch_size PATCH_SIZE] [--test_batch_size TEST_BATCH_SIZE]
               [--loss LOSS] [--hdelta HDELTA] [--change_weight]
               [--num_pyramid NUM_PYRAMID] [--pyramid_loss]
               [--filter_size FILTER_SIZE] [--re_load] [--load_best]
               [--model_name MODEL_NAME] [--file_name FILE_NAME]
               [--save_path SAVE_PATH]

and in this case I used the following command:
python C:\development\python\depth_map\Guided-Depth-Map-Super-resolution-A-Survey\code\main.py  --cpu CPU --scale=8 --dataset_name='NYU' --model_name=dgf

but I receive same error:
1.Train
'nvidia-smi' is not recognized as an internal or external command,
operable program or batch file.
